test/unit/test_auth_keypair.py

```diff
@@ -106,21 +110,34 @@ class Bad:
     for bad_private_key in ("abcd", 1234, Bad()):
         auth_instance = AuthByKeyPair(private_key=bad_private_key)
         with raises(TypeError) as ex:
-            auth_instance.handle_timeout(
-                authenticator="SNOWFLAKE_JWT",
-                service_name=None,
-                account=account,
-                user=user,
-                password=None,
-            )
+            auth_instance.prepare(account=account, user=user)
```

Comment:
handle_timeout won't throw TypeError now?
I think `handle_timeout` calls the `prepare` in `AuthByKeyPair right?